spec/parser_spec.rb

```diff
@@ -11,15 +11,16 @@
 
   describe '#read_first_line' do
     it 'matches string in file' do
-      data = Parser.new('games.log')
+      data = Parser.new('spec/fixture/games_test.log')
       expect(data.read_first_line).to match("  0:00 ------------------------------------------------------------\n")
     end
   end
 
   describe '#log_json' do
     it 'returns correct output' do
-      data = Parser.new('games.log')
-      expect(data.log_json).to match('{\"games.log\":{\"lines\":5306}}')
+      data = Parser.new('spec/fixture/games_test.log')
+      expect(data.log_json.to_json).to match("{\"games.log\":{\"lines\":158,\"players\":" +
```

Comment:
```suggestion
      expect(data.log_json).to match("{\"games.log\":{\"lines\":158,\"players\":" +
```

Você não precisa converter o resultado aqui para JSON